### PR TITLE
TypeScript to LiveScript

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by AyaMorisawa** [comment]
https://github.com/afuafu55/Misskey/blob/5b96807c777d3e60c576b4f030ea242d5ce08222/src/api/rest/search/user.ls#L20
cannot use semicolon

**Comment by AyaMorisawa** [comment]
also
https://github.com/afuafu55/Misskey/blob/5b96807c777d3e60c576b4f030ea242d5ce08222/src/api/rest/search/user.ls#L25

File: src/api/rest/search/user.ls [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require! {
+	async
+	'../../api-response': APIResponse
+	'../../../models/application': Application
+	'../../../models/User': User
+	'../../../utils/streaming': Streamer
+}
+
+authorize = require '../../auth'
```
Comment (syuilo) [inline src/api/rest/search/user.ls:9]: whats?

**Comment by ghost** [comment]
I'm Sorry.

**Comment by AyaMorisawa** [comment]
already fixed
don't worry

**Comment by ghost** [comment]
Thank you.

**Pull request merged**